content/en/docs/setup/independent/high-availability.md

```diff
@@ -129,23 +153,24 @@ will fail the health check until the kube-apiserver is running.
               - CP0_HOSTNAME
               - CP0_IP
         networking:
-            # This CIDR is a calico default. Substitute or remove for your CNI provider.
+            # This CIDR is a Calico default. Substitute or remove for your CNI provider.
             podSubnet: "192.168.0.0/16"
+        
 
-1. Replace the following variables in the template that was just created with
-   values for your specific situation:
+1.  Replace the following variables in the template with the appropriate
```

Comment:
oh my gracious this extra space after the number is a blackfriday thing isn't it?

Reply:
i think yes. blackfriday has(had?) bugs with the numbered lists, that are fixed upstream, but `k/website` hasn't updated yet, AFAIK.

Reply:
I prefer that extra space, actually. It means the words in the list item line up on a tab stop, and makes it easier to see how much to indent interior block-level content like code blocks (or in Black Friday, additional paragraphs in the list item).